### ISSUE TITLE
set "type": "module" in the package.json

### ISSUE THREAD

**Issue by LeonardoRick** [nuxt-modules/i18n#2213]
### Environment

- Operating System: `Darwin`
- Node Version:     `v18.16.0`
- Nuxt Version:     `3.6.2`
- Nitro Version:    `2.5.2`
- Package Manager:  `npm@9.7.1`
- Builder:          `vite`
- User Config:      `devtools`, `css`, `routeRules`, `runtimeConfig`, `modules`, `pinia`, `app`, `nitro`
- Runtime Modules:  `@nuxtjs/tailwindcss@6.8.0`, `@pinia/nuxt@0.4.11`, `@nuxtjs/i18n@7.3.1`, `@vue-macros/nuxt@1.3.12`, `@nuxtjs/fontaine@0.4.0`
- Build Modules:    `-`

### Reproduction

https://github.com/LeonardoRick/temp-nuxti18n-bug

### Describe the bug

Running the latest version of `nuxt` and `i18n` library causes the error  `ERROR  (node:36792) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension. `

As you can see on reproduction [package.json](https://github.com/LeonardoRick/temp-nuxti18n-bug/blob/main/package.json), `"type": "module"` is settle.

This bug as already [raised on `nuxt` side ](https://github.com/nuxt/nuxt/issues/20895)and it seems to be a `i18n` issue

### Logs

```shell
ERROR  (node:36792) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.                                                                                                       4:25:38 PM
(Use node --trace-warnings ... to show where the warning was created)


 ERROR  Cannot start nuxt:  Cannot read properties of undefined (reading 'options')                                                                                                                                           4:25:38 PM

  at _default (/Users/leonardorick/node_modules/@nuxtjs/i18n/src/index.js:15:92)
  at installModule (node_modules/@nuxt/kit/dist/index.mjs:2409:101)                                                                                                                                                           4:25:36 PM
  at async initNuxt (node_modules/nuxt/dist/index.mjs:3231:7)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:205:9)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:249:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:49:20)
```

**Comment by kazupon** [comment]
Thank you for your reporting!

I checked your reproduction, however I could not see this issue,
so your reproduction works
`"type": "module"` has already been resolved in v8 beta 13 and is no longer required.

I’ve confuese by your reporting description

> @nuxtjs/i18n@7.3.1

Your description of PR mentions nuxt i18n module v7.
Your reproduction code is nuxt i18n module v8. nuxt i18n module v7 does not support Nuxt 3.

**Comment by LeonardoRick** [comment]
@kazupon thank you so much for your attention on the logs. I realise I'm not being able to update this version properly. I already ran `rm -rf node_modules package-lock.json && npx nuxi cleanup`. I even tried to create a new project and the `runtime module` version that appears is still `@nuxtjs/i18n@7.3.1`, even on `package.json` and `node_modules` its being the new one `8.0.0-beta.13` I even tried to create a brand new project  and the version sill shows the old one. Which is really weird.

Am I missing something?

I thought it could be related to some global packages installed but I'm setting everything up with `npx` so Idk what else could it be

**Comment by kazupon** [comment]
Thank you for your detail information!

I've tried your reproduction again, but I could not reproduce.
`"type": "module"` issue has been fixed in v8.0.0-beta.13
If you will see the this issue again, you will open the issue as new.

so close.

Thanks!
